### ISSUE TITLE
[Package Issue]: Streetwriters.Notesnook should default to normal install (not portable)

### ISSUE THREAD

**Issue by JVimes** [microsoft/winget-pkgs#125309]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

This is a traditional desktop app, but currently there's no start menu shortcut because the package picks the Portable installer by default.

I think it would make sense to install the non-portable (`--installer-type nullsoft`) type by default.

### Steps to reproduce

`winget install -e --id Streetwriters.Notesnook`

### Actual behavior

It runs `notesnook_win_x64_portable.exe`

### Expected behavior

It runs `notesnook_win_x64.exe`. This  can currently can be done with `--installer-type nullsoft`.

### Environment

```raw
Windows Package Manager (Preview) v1.7.2782-preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.3636
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.22.2782.0
```

**Comment by mdanish-kh** [comment]
This should be fixed now through #125321. Can you please check again?

**Comment by JVimes** [comment]
Works great, thank you @mdanish-kh!
